rpc/common/moc_common_nodeinfo.proto

```diff
@@ -33,10 +33,12 @@ message PhysicalMemory {
 	uint64 sizeBytes = 1;
 }
 
-message GPU {
+message HostGPU {
 	string name = 1;
 	uint32 sizeBytes = 2;
-	uint32 count = 3;
+	uint64 partitionSizeBytes = 3;
```

Comment:
This would break the protobuf contract?
Any removal of params should not reuse the field id. We need to call it out as deprecated and use new field